packages/connect/src/transport.ts

```diff
@@ -50,6 +50,6 @@ export interface Transport {
     signal: AbortSignal | undefined,
     timeoutMs: number | undefined,
     header: HeadersInit | undefined,
-    input: AsyncIterable<I>
+    input: AsyncIterable<PartialMessage<I>>
```

Comment:
A breaking change for implementors of `Transport`, but stream() should have accepted partial messages in the first place.